dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -1,24 +1,68 @@
-import type { DCRContainerPalette } from '../../types/front';
+import type { DCRContainerPalette, DCRGroupedTrails } from '../../types/front';
 import { LI } from './Card/components/LI';
 import { UL } from './Card/components/UL';
 import { FrontCard } from './FrontCard';
 
 type Props = {
-	trails: TrailType[];
+	groupedTrails: DCRGroupedTrails;
 	containerPalette?: DCRContainerPalette;
 	showAge?: boolean;
 };
 
+const SnapRow = ({
+	snap,
+	containerPalette,
+	showAge,
+}: {
+	snap: TrailType;
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+}) => {
+	return (
+		<UL direction="row">
+			<LI padSides={true} padBottom={true}>
+				<FrontCard
+					trail={snap}
+					starRating={snap.starRating}
+					containerPalette={containerPalette}
+					containerType="dynamic/package"
+					showAge={showAge}
+					headlineSize="large"
+					headlineSizeOnMobile="medium"
+					imagePosition="right"
+					imagePositionOnMobile="left"
+					imageSize="medium"
+					trailText={snap.trailText}
+				/>
+			</LI>
+		</UL>
+	);
+};
+
 export const DynamicPackage = ({
-	trails,
+	groupedTrails,
 	containerPalette,
 	showAge,
 }: Props) => {
-	const [primary, ...remaining] = trails;
+	// Take the first 'snap' - all others are treated as standards
+	const snap = groupedTrails.snap.shift();
+	const [primary, ...remaining] = [
+		...groupedTrails.snap,
+		...groupedTrails.standard,
+	];
 
+	// TODO: Different layouts are required when there are 4+ 'remaining' cards
+	// dynamic/package will display up to 8 'remaining' cards before moving to 'show more'
 	if (primary.isBoosted) {
 		return (
 			<>
+				{!!snap && (
```

Comment:
I think this is probably fine for now and this is one for a later PR but I think this logic (er, that I wrote) could be improved by separating out the decision logic for what gets shown into a pre render step

Reply:
Yeah definitely - I decided I wouldn't refactor the rendering logic for this PR, and leave that for a later one (where we cover the edge cases the container currently misses)